src/WebJobs.Script.WebHost/Security/ScriptSecrets.cs

```diff
@@ -15,6 +17,9 @@ public abstract class ScriptSecrets
     {
         protected ScriptSecrets()
         {
+            GeneratedBy = string.Format(CultureInfo.InvariantCulture, "HostName={0}, InstanceId={1}",
```

Comment:
This information does not show the _system_ that generated the file. If we were to add the same process to Kudu, we'd end up with the same data. You had a property indicating this had been generated by the runtime, can you keep that? Also, I think we'd be better off with distinct properties instead of `HostName=host, InstanceId={instance}`. You don't need a complex object, but just distinct properties.